frontend/src/components/map/map.js

```diff
@@ -154,18 +173,25 @@ class Map extends Component {
             )}
             {/* Plots existing reports onto the map */}
             {this.props.reports.map((report) => {
-              return (
-                <ReportContainer
-                  key={report._id}
-                  report = {report}
-                  icon= {{
-                    url: "./toilet-paper.svg",
-                    scaledSize: new window.google.maps.Size(40, 40),
-                  }}
-                  labelAnchor={new window.google.maps.Point(-5, 50)}
-                />
-              );
-            })}
+              // Milliseconds elapsed since the UNIX epoch represented as integer
+              const currentDateTime = Date.now();
+              const reportDateTime = Date.parse(report.date);
+              // millisecondsDiff / 1000 => seconds; seconds / 60 => minutes; minutes / 60 => hours
+              const millisecondsDiff = Math.floor((currentDateTime - reportDateTime) / 1000 / 60 / 60);
+              if (millisecondsDiff <= this.state.timeFilter) {
```

Comment:
Semantic changes for more code readability. `diff` variable renamed to `millisecondsDiff`.